flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/factory/KubernetesJobManagerFactoryTest.java

```diff
@@ -69,6 +71,9 @@ public void setup() throws Exception {
 		flinkConfig.set(KubernetesConfigOptionsInternal.ENTRY_POINT_CLASS, ENTRY_POINT_CLASS);
 		flinkConfig.set(KubernetesConfigOptions.JOB_MANAGER_SERVICE_ACCOUNT, SERVICE_ACCOUNT_NAME);
 
+		setHadoopConfDirEnv();
```

Comment:
We might possibly have a subclass of this test class so that don't pollute global settings. Also it seems we don't test existing hadoop conf case and empty local hadoop conf case.

Reply:
Goo catch. Comments addressed via [64eaf71](https://github.com/apache/flink/pull/11415/commits/64eaf715bef2bfc9492ef197ecec62d3e067d18d)